### PR TITLE
Celsius for external sessions

### PR DESCRIPTION
**Pull request by anna1901**
# What was done

This PR fixes displaying right values and colors then user uses C instead of F
- [x]  S&F completion screen
- [x]  S&F chart
- [x]  value, sensorName, border, and dot in StaticSingleStreamView
- [x]  updating ThresholdsSettingsView for External Card
- [x]  showing right values on graph

https://trello.com/c/RMJKVz91

### CODE REVIEW

**Comment by anna1901** [comment]
[Updating Celsius thresholds for External Sessions](https://trello.com/c/RMJKVz91/682-updating-celsius-thresholds-for-external-sessions)

File: AirCasting/SessionViews/SessionCardView.swift [hunk 1]
```diff
@@ -155,7 +155,6 @@ private extension SessionCardView {
     private var graphButton: some View {
         Button {
             isGraphButtonActive = true
-            Log.info("\(reorderButton)")
```
Comment (jankrzempek) [inline AirCasting/SessionViews/SessionCardView.swift:158]: 🙌🏻

File: AirCasting/SearchAndFollow/CompleteScreen/Views/CompleteScreenViewModel.swift [hunk 2]
```diff
@@ -210,9 +214,9 @@ class CompleteScreenViewModel: ObservableObject {
             
             self.sessionStreams = .ready(self.externalSessionWithStreams!.streams.map {
                 .init(id: $0.id,
-                      sensorName: Self.getSensorName($0.sensorName),
+                      sensorName: self.getShortSensorName($0.sensorName),
                       sensorUnit: $0.unitSymbol,
-                      lastMeasurementValue: $0.measurements.last?.value,
+                      lastMeasurementValue: $0.measurements.last?.value != nil ? self.showConvertedValue($0.measurements.last!.value, sensorName: $0.sensorName) : $0.measurements.last?.value,
```
Comment (jankrzempek) [inline AirCasting/SearchAndFollow/CompleteScreen/Views/CompleteScreenViewModel.swift:219]: This is a bit weird for me, I mean: you are checking if the value is not nil and after that, if it is - you are using it.

**Review by jankrzempek — APPROVED** [review]
Works like a charm 👌🏻

**Pull request merged**